certified-connectors/VEV-IQ/Readme.md

```diff
@@ -0,0 +1,88 @@
+
+Enter Request Body:
+Click on the "Body" tab below the headers section. Choose the "raw" option and set the format to "JSON (application/json)".
+
+Paste the following JSON request body into the input field:
+{
+  "email": "api.user@domain.com",
+  "password": "Your-API-Password",
+  "acceptEula": true,
+  "tenant": "Your-Tenant-Name"
+}
+
+Replace "api.user@domain.com", "Your-API-Password", and "Your-Tenant-Name" with your actual credentials and tenant information.
+
+Send the Request:
+Click the "Send" button to submit the request to the API.
+
+Retrieve Bearer Token:
+Once the request is sent successfully, you'll receive a response from the API. Look for the "token" field in the response body. This is the bearer token you need to authenticate subsequent requests.
+
+Use Bearer Token:
+Copy the bearer token from the response. You can now use this token as an "Authorization" header in your subsequent API requests.
+
+Remember that you can retrieve bearer tokens eonly if you have a dedicated API user in the VEV-IQ environment. Please contact your administrator to do so.
+
+This process allows you to retrieve a bearer token using Postman, which can then be used to securely access the API endpoints in your custom connector.
+
+
+## Supported Operations
+The connector supports the following operations:
+* `List sites`: Lists all the sites in your tenant
+* `List charging stations`: Lists all the charging stations in your tenant
+* `Get transactions`: Gets all transactions in your tenant
+* `Get completed transactions`: Gets all completed transactions in your tenant
+* `Get active transactions`: Gets all active transactions in your tenant
+* `List users`: Lists all users in your tenant
+* `List tags`: Lists all tags in your tenant
+* `List assets`: Lists all assets in your tenant
+* `List car catalogs`: List all car catalogs in your tenant
+* `List car makers`: Lists all car makers in your tenant
+* `List cars`: Lists all cars on your tenant
+* `List site areas`: Lists all site areas in your tenant
+* `Get statistics charging stations consumption`: Gets statistics charging stations consumption in your tenant
+* `Get statistics charging stations inactivity`: Gets statistics charging stations inactivity in your tenant
+* `Get statistics charging stations usage`: Gets statistics charging stations usage in your tenant
+* `Get statistics charging station transaction`: Gets statistics charging stations transactions in your tenant
+* `Get statistics users consumption`: Gets statistics users consumption in your tenant
+* `Get statistics users inactivity`: Gets statistics users inactivity in your tenant
+* `Get statistics users usage`: Gets statistics users usage in your tenant
+* `Get statistics users transaction`: Gets statistics users transactions in your tenant
+* `Get statistics users pricing`: Gets statistics users pricing in your tenant
+* `Get invoices`: Gets invoices information from your tenant
+
```

Comment:
Please add below sections in Readme.md file. 
1. Obtaining credentials 
2. Known issues and limitations
3. Deployment instructions. 

Please refer this link - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Done! thank you